docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -0,0 +1,231 @@
+[[role="xpack"]]
+[[drilldowns]]
+== Make dashboards interactive
+
+Add interactive capabilities to your dashboard, such as controls that allow you to apply dashboard-level filtes, and drilldowns that allow you to navigate to other dashboards and external websites. 
+
+++++
+<script type="text/javascript" async 
+src="https://play.vidyard.com/embed/v4.js"></script>
+<img
+style="width: 100%; margin: auto; display: block;"
+class="vidyard-player-embed"
+src="https://play.vidyard.com/UhGkdJGC32HRn3oS5ZYJL1.jpg"
+data-uuid="UhGkdJGC32HRn3oS5ZYJL1"
+data-v="4"
+data-type="inline"
+/>
+</br>
+++++
+
+[float]
+[[add-controls]]
+== Add control panels
+
+*Controls* panels allow you to apply dashboard-level filters based on values from a list, or a range of values. 
+
+{kib} supports two types of *Controls*:
+
+* *Options list* — Filters content based on one or more specified options. The dropdown menu is dynamically populated with the results of a terms aggregation. 
+For example, use the options list on the sample flight dashboard when you want to filter the data by origin city and destination city.
+
+* *Range slider* — Filters data within a specified range of numbers. The minimum and maximum values are dynamically populated with the results of a 
+min and max aggregation. For example, use the range slider when you want to filter the sample flight dashboard by a specific average ticket price.
++
+[role="screenshot"]
+image::images/dashboard-controls.png[]
+
+. On the dashboard, click *All types*, then select *Controls*.
+
+. Click *Options*, then configure the following options:
+
+* *Update {kib} filters on each change* &mdash; When selected, all interactive inputs create filters that refresh the dashboard. When unselected,
+ {kib} filters are created only when you click *Apply changes*.
+
+* *Use time filter* &mdash; When selected, the aggregations that generate the options list and time range are connected to the <<set-time-filter,time filter>>.
+
+* *Pin filters for all applications* &mdash; When selected, all filters created by interacting with the inputs are automatically pinned.
+
+. Click *Update*
+
+[float]
+[[create-drilldowns]]
+== Create drilldowns
+
+When you interact with the values and in a visualization panel, {kib} creates and applies a filter for the value to the entire dashboard. To customize the behavior of what happens when you click a value, create drilldowns. Drilldows are shortcuts that allow you to quickly navigate between your most used dashboards, or navigate to external websites from your dashboards. You can create multiple drilldowns for a single panel. 
```

Comment:
It's not very specific to say "When you interact with the values", since we have seen confusion about how this works. It's also not just clicks, for example you can drag to select a range of values in some chart types.

```suggestion
Each panel type in {kib} has a built-in interaction which applies a filter to the dashboard, for example by dragging a time range or clicking a pie slice. Drilldowns let you customize this behavior while keeping the context of the interaction. For example, if you add a drilldown to a Lens panel you navigate from a summary dashboard to a dashboard with a filter for the specific host name you filtered. You could also navigate to an external website with the specific host name as a parameter. You can create multiple drilldowns for a single panel.
```